community/qt5-qtbase/APKBUILD

```diff
@@ -7,13 +7,14 @@ _ver=${_ver/_/-}
 _ver=${_ver/beta0/beta}
 _ver=${_ver/rc0/rc}
 _V=${_ver/rc/RC}
-pkgrel=0
+pkgrel=1
 pkgdesc="Qt5 - QtBase components"
 url="http://qt-project.org/"
 arch="all"
 license="LGPL-2.0 with exceptions or GPL-3.0 with exceptions"
```

Comment:
those are not SPDX identifiers but whatever

Reply:
I _think_ I corrected them properly, but someone with more knowledge of this will have to verify.